### DISCUSSION TITLE
Müller Licht 44435 identified as Müller Licht 404028

### DISCUSSION THREAD

**Discussion by ovizii** [Koenkk/zigbee2mqtt#15468] [Devices]
Since the 2 devices expose exactly the same features I don't really mind, just wanted to mention it somewhere.

https://www.zigbee2mqtt.io/devices/44435.html 
vs 
https://www.zigbee2mqtt.io/devices/404028.html

![image](https://user-images.githubusercontent.com/837902/206780411-e31557a4-41bc-44f1-8cc0-06a0d7b6c98e.png)

**Comment by Koenkk** [comment]
These devices use the same modelID, renamed it to `404028/44435`
